src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1059,4 +1063,55 @@ private void sendEvent(EventType eventType, EventPayload payload, Object caseId)
         .with("transactionId", transactionId)
         .debug("{} event published", eventType);
   }
+
+  private CaseDTO getLatestCaseFromRmOrCache(UUID caseId, Boolean getCaseEvents)
```

Comment:
There is an existing method that does same but by uprn.
When you put these two method names side by side it unclear which does what.
They should both be renamed to make this clearer.
Suggest :

getLatestCaseById()
getLatestCaseByUPRN()

Reply:
that makes sense - done